knip.jsonc

```diff
@@ -0,0 +1,46 @@
+{
+    "workspaces": {
+        ".": {
+            // These entries are complementary to the ones found in package.json
+            "entry": [
+                "lib/rules/index.js",
+                "tools/internal-rules/index.js",
+                "tools/update-rule-types.js",
+                "wdio.conf.js" // Remove when Knip has a wdio plugin
+            ],
+            "project": ["{conf,lib,tools}/**/*.js"],
+            "mocha": {
+                "entry": [
+                    "tests/{bin,conf,lib,tools}/**/*.js", // see Makefile.js
+                    "tests/_utils/test-lazy-loading-rules.js"
+                ],
+                "project": ["tests/**/*.js"]
+            },
+            "ignore": [
+                // If Knip would consider exports as named, their usage is too dynamic: globals[`es${ecmaVersion}`]
+                // An alternative is to add `__esModule: true` to the export and we can remove it here from the ignores:
+                "conf/globals.js",
+                // These contain unresolved imports and other oddities:
+                "tests/bench/large.js",
+                "tests/lib/rule-tester/rule-tester.js",
+                "tests/performance/jshint.js",
```

Comment:
[Question] What are those unresolved imports and other oddities, specifically?

Reply:
You can see the difference by removing (any of) these lines in the configuration, and running `npm run lint:knip`